package.json

```diff
@@ -32,6 +31,9 @@
     "typedoc": "0.22.13",
     "typescript": "4.1.2"
   },
+  "peerDependencies": {
+    "axios": "^1.6.8"
```

Comment:
I see that `sdk-dapp` has another constraint:

```
"axios": ">=1.6.5",
```

https://github.com/multiversx/mx-sdk-dapp/blob/development/package.json#L128

But it's fine `^1.6.8` :+1: